static/components/dashboard/sources-countries.js

```diff
@@ -42,34 +47,37 @@ customElements.define(
                 icon: "bing.com"
             },
             "Reddit": {
-                match: ["reddit.com", ".reddit.com"],
+                match: ["reddit.com", ".reddit.com",
+                    "com.laurencedawson.reddit_sync", "com.laurencedawson.reddit_sync.pro",
+                    "com.andrewshu.android.reddit".
```

Comment:
Ha, apologies, I was certain I added such a comment before: Can you replace the dot with a comma.

Reply:
Alright, I... think I did that now. I'm using Github's web UI which... isn't the best way to do this, but I don't have a functional CLI setup right now and I'm not that great at git in the first place.

Reply:
No worries.